@here/harp-mapview/lib/MapView.ts

```diff
@@ -2718,15 +2722,25 @@ export class MapView extends THREE.EventDispatcher {
         this.m_screenCollisions.update(width, height);
 
         this.m_pixelToWorld = undefined;
-        this.m_fog.update(this, this.m_viewRanges.maximum);
+        this.m_fog.update(this, this.m_viewState.viewRanges.maximum);
 
-        this.updateLookAtSettings();
+        this.updateViewState();
```

Comment:
This is still a bit messy. We update the `ViewState` partially before we call `updateViewState`. Eventually everything should be updated in `updateViewState`